EIPS/eip-1.md

```diff
@@ -232,6 +232,64 @@ Permitted Networking Specifications URLs must anchor to a specific commit, and s
 ^https://github.com/ethereum/devp2p/blob/[0-9a-f]{40}/.*$
 ```
 
+### Other Well Known Standards Bodies
+
+Links to other well known standards bodies specification may be included using normal markdown syntax. Be mindful of the requirements set for each SDO as these were chosen in order to maintain links to stable URLs.
+
+#### World Wide Web Consortium (W3C)
+
+Links to a W3C "Recommendation" status specification may be included using normal markdown syntax, where the content of the link is the title of the specification. For example, the following link would be allowed:
+
+```markdown
+[W3C Secure Context](https://www.w3.org/TR/secure-contexts/)
+```
+
+Which renders as:
+
+[W3C Secure Context](https://www.w3.org/TR/secure-contexts/)
+
+Permitted W3C recommendation URLs MUST anchor to a specification in the technical reports namespace, and so MUST match this regular expression:
+
+```regex
+^https:\/\/www.w3.org\/TR\/.*$
+```
+
+#### Internet Engineering Task Force (IETF)
+
+Links to an Request For Comment (RFC) specification may be included using normal markdown syntax, and must follow the non-hyphenated format `RFC X`, where `X` is the RFC number. For example, the following link would be allowed:
+
+```markdown
+[RFC 8446](https://www.rfc-editor.org/rfc/rfc8446)
+```
+
+Which renders as:
+
+[RFC 8446](https://www.rfc-editor.org/rfc/rfc8446)
+
+Permitted IETF specification URLs MUST anchor to a specification with an assigned RFC number (meaning cannot reference internet drafts), and so MUST match this regular expression:
+
+```regex
+^https:\/\/www.rfc-editor.org\/rfc\/.*$
+```
+
+#### Web Hypertext Application Technology Working Group (WHATWG)
+
+Links to WHATWG specifications may be included using normal markdown syntax, where the content of the link is the title of the specification, followed by ` Specification` if applicable. For example, the following link would be allowed:
+
+```markdown
+[WHATWG HTML Spec](https://html.spec.whatwg.org/)
```

Comment:
We prefer links to exact versions:

```suggestion
[WHATWG HTML Spec](https://html.spec.whatwg.org/commit-snapshots/578def68a9735a1e36610a6789245ddfc13d24e0/)
```

With the commit, the reader is aware of exactly where the living standard was when the EIP was finalized. The reader can then choose exactly how to implement the EIP: either as written, or with the latest spec from WHATWG.